fineract-provider/build.gradle

```diff
@@ -398,123 +398,127 @@ configurations.driver.each {File file ->
+        classpath 'mysql:mysql-connector-java:8.0.12'
     }
 }
 
 
 flyway {
     url = "jdbc:mysql:thin://localhost:3306/mifostenant-default"
-	driver = "org.drizzle.jdbc.DrizzleDriver"
+	driver = "com.mysql.cj.jdbc.Driver"
     user = mysqlUser
     password = mysqlPassword
 }
 
-task migrateTenantDB<<{
-    description="Migrates a Tenant DB. Optionally can pass dbName. Defaults to 'mifostenant-default' (Example: -PdbName=someTenantDBname)"
 
-	def filePath = "filesystem:$projectDir" + '/src/main/resources/sql/migrations/core_db'
-	def tenantDbName = 'mifostenant-default';
+task migrateTenantDB(type: org.flywaydb.gradle.task.FlywayMigrateTask) {
+    description="Migrates a Tenant DB. Optionally can pass dbName. Defaults to 'mifostenant-default' (Example: -PdbName=someTenantDBname)"
+    def filePath = "filesystem:$projectDir" + '/src/main/resources/sql/migrations/core_db'
+    def tenantsDbName = 'mifostenant-default';
     if (rootProject.hasProperty("dbName")) {
-		tenantDbName = rootProject.getProperty("dbName")
-	}
+        tenantsDbName = rootProject.getProperty("dbName")
+    }
 
-    flyway.url= "jdbc:mysql:thin://localhost:3306/$tenantDbName"
-	flyway.driver = "org.drizzle.jdbc.DrizzleDriver"
+    flyway.url= "jdbc:mysql:thin://localhost:3306/$tenantsDbName?useSSL=false"
+    flyway.driver = "com.mysql.cj.jdbc.Driver"
+    flyway.user = 'root'
+    flyway.password = 'mysql'
     flyway.locations= [filePath]
-    /**We use ${ as the prefix for strecthy reporting, do not want them to be interpreted by Flyway**/
-    flyway.placeholderPrefix = "\$\${"
-    flywayMigrate.execute()
 }
 
-task showTenantDBInfo<<{
-    description="Shows the migration info for a Tenant DB. Optionally can pass dbName. Defaults to 'mifostenant-default' (Example: -PdbName=someTenantDBname)"
 
-	def filePath = "filesystem:$projectDir" + '/src/main/resources/sql/migrations/core_db'
+task showTenantDBInfo(type: org.flywaydb.gradle.task.FlywayMigrateTask){
+    description="Shows the migration info for a Tenant DB. Optionally can pass dbName. Defaults to 'mifostenant-default' (Example: -PdbName=someTenantDBname)"
+    def filePath = "filesystem:$projectDir" + '/src/main/resources/sql/migrations/core_db'
     def tenantDbName = 'mifostenant-default';
     if (rootProject.hasProperty("dbName")) {
-		tenantDbName = rootProject.getProperty("dbName")
-	}
+        tenantDbName = rootProject.getProperty("dbName")
+    }
 
-    flyway.url= "jdbc:mysql:thin://localhost:3306/$tenantDbName"
-	flyway.driver = "org.drizzle.jdbc.DrizzleDriver"
+    flyway.url= "jdbc:mysql:thin://localhost:3306/$tenantDbName?useSSL=false"
+    flyway.driver = "com.mysql.cj.jdbc.Driver"
     flyway.locations= [filePath]
-    flywayInfo.execute()
 }
 
 
-task migrateTenantListDB<<{
+task migrateTenantListDB(type: org.flywaydb.gradle.task.FlywayMigrateTask) {
```

Comment:
@michaelbukachi On running  `./gradlew migrateTenantListDB -PdbName=mifosplatform-tenants` which gets this executed, i had the following warning 

`Fri Mar 15 21:42:41 WAT 2019 WARN: Establishing SSL connection without server's identity verification is not recommended. According to MySQL 5.5.45+, 5.6.26+ and 5.7.6+ requirements SSL connection must be established by default if explicit option isn't set. For compliance with existing applications not using SSL the` `verifyServerCertificate property is set to 'false'. You need either to explicitly disable SSL by setting useSSL=false, or set useSSL=true and provide truststore for server certificate verification.`

so i had to use this `jdbc:mysql:thin://localhost:3306/$tenantsDbName?useSSL=false"` instead so as to allow the SSL.
Now i get this warning that 
`DB: Changing sql mode 'NO_AUTO_CREATE_USER' is deprecated. It will be removed in a future release. (SQL State: HY000 - Error Code: 3090)`
`DB: Changing sql mode 'NO_AUTO_CREATE_USER' is deprecated. It will be removed in a future release. (SQL State: HY000 - Error Code: 3090)`

Reply:
How ever this writes to the database the tables. I the ran this  `./gradlew migrateTenantDB -PdbName=mifostenant-default` of which i expect it to migrate => `task migrateTenantDB(type: org.flywaydb.gradle.task.FlywayMigrateTask)`  but it does the same thing as above and writes the table of mifosplatform-tenants to mifostenant-default. I am a little confused on the behaviour. 

Similary when i just run `./gradlew flywayMigrate` it writes to the 2 db the same info. What could i be doing wrong?